app/controllers/admin/domain_allows_controller.rb

```diff
@@ -28,6 +32,33 @@ def destroy
     redirect_to admin_instances_path, notice: I18n.t('admin.domain_allows.destroyed_msg')
   end
 
+  def export
+    authorize :domain_allow, :create?
```

Comment:
It might make sense to use `authorize :instance, :index?` or create a new policy here (although in practice all those policies are currently equivalent to just checking the logged in user is an admin).